settings/remarks/microsoft.cache/remarks.json

```diff
@@ -29,7 +29,7 @@
     {
       "ResourceType": "Microsoft.Cache/redisEnterprise",
       "Path": "samples/redisenterprise/main.tf",
-      "Description": "A basic example of deploying Redis Enterprise Cluster."
+      "Description": "A basic example of deploying Redis Enterprise Cluster.\n\nNote: Redis Enterprise (Microsoft.Cache/redisEnterprise) is also known as Azure Managed Redis. It requires a Microsoft.Cache/redisEnterprise/databases child resource to function properly. See https://learn.microsoft.com/azure/redis/overview for more information."
```

Comment:
Could you break out the general resource info / deprecation notes into a separate PR? We have a specific place to put them, separate from the samples - see here for example:

https://github.com/Azure/template-reference-generator/blob/1979a35951c1532412c65feefb357cc74b3614dc/settings/remarks/microsoft.authorization/remarks.json#L3-L11

There's some info on the dfiferent fields written up here: https://github.com/Azure/template-reference-generator/blob/main/docs/configuration.md#remarks